src/rules/css/detectBadMozBinding.js

```diff
@@ -2,16 +2,17 @@ import * as messages from 'messages';
 import { LOCAL_CSS_URL } from 'regex';
 
 
-export function detectBadMozBindingURL(rule) {
+export function detectBadMozBindingURL(cssRule, filename= null,
+                                       {startLine, startColumn}={}) {
   var messageList = [];
-  for (let declaration of rule.declarations) {
+  for (let declaration of cssRule.declarations) {
```

Comment:
Won't this add extra confusion because the _rule_ here is the argument being passed and the _instruction_ is the actual rule function? Or am I mis-understanding your previous comment?